atlasdb-config/src/main/java/com/palantir/atlasdb/factory/Refreshables.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * (c) Copyright 2020 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.atlasdb.factory;
+
+
+import java.time.Duration;
+import java.util.concurrent.Callable;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.TimeUnit;
+import java.util.function.Consumer;
+
+import com.palantir.logsafe.Preconditions;
+import com.palantir.logsafe.exceptions.SafeRuntimeException;
+import com.palantir.refreshable.DefaultRefreshable;
+import com.palantir.refreshable.Refreshable;
+
+final class Refreshables {
```

Comment:
Can you push this class into the refreshables repo please? I think I will need the same code in internal ski product.

Reply:
@iamdanfox you felt strongly about removing this method in the first place, what do you think?

Reply:
this is only used here to adapt from a supplier to a refreshable right? I don't think you need this in internal ski product.

Reply:
no ski product would not need this, but any library that would like to create refreshables would.

Reply:
I think I actually do feel strongly about keeping this out. I don't want open-sourcing refreshable to mean that we find a bazillion usages of refreshable everywhere, I'm ok with there being a little friction in creating entirely new ones, as I still want them to _primarily_ come from WC.

Reply:
Ack, I forgot we can adapt the WC one.